client/src/screens/groups/Index.js

```diff
@@ -116,11 +118,17 @@ Group.propTypes = {
   }),
 };
 
-class Groups extends Component {
-  static navigationOptions = {
+class Index extends Component {
```

Comment:
pretty sure the idea of Index is supposed to be a default for this directory or whatever. I don't really agree with naming a component `Index`. cc @sjmcculloch

Reply:
Also this name isn't actually used outside of this file because you export default the decorated result at the bottom